api/service.capnp

```diff
@@ -0,0 +1,54 @@
+using Go = import "/go.capnp";
+
+@0xfcba4f486a351ac3;
+
+$Go.package("discovery");
+$Go.import("github.com/wetware/ww/internal/api/service");
+
+
+interface ServiceDiscovery {
```

Comment:
What do you think of calling this `Directory`?  The imported symbol name would be `service.Registry`, which I think identifies its function rather well.   In particular, it avoids the confusion/stuttering of `service.ServiceDiscovery` that we've been struggling with.

Reply:
`service.Registry` is another possibility

Reply:
Do you mean `Directory` or `Registry`? I would go with `Registry`